modules/web-mercator/test/spec/versus-mapbox/versus-mapbox.spec.js

```diff
@@ -25,6 +26,7 @@ test('Viewport vs Mapbox project', t => {
 
   for (const viewportName in VIEWPORT_PROPS) {
     const viewportProps = VIEWPORT_PROPS[viewportName];
+    if (viewportProps.pitch > 60) continue;
```

Comment:
Nits:
- We generally don't use continue, but fine. (Though it seems like since almost all loops skip this so maybe could have just split the test cases in two separate test case arrays instead.)
- Should 60 be a constant?


More importantly, why doesn't our linter demand braces here after if statement? That looks like a bigger change of style.

Reply:
+1